content/api/subaccounts.apib

```diff
@@ -43,6 +45,19 @@ When making POST, PUT, or DELETE requests without the `X-MSYS-SUBACCOUNT` header
 
 <Banner status="info">`X-MSYS-SUBACCOUNT` is not required, but if provided, must be a number.</Banner>
 
+### Pagination
+The Subaccounts API supports pagination for queries with larger result sets. Pagination is done through the use of the `cursor` and `per_page` query parameters.
+The `per_page` query parameter tells the Subaccounts API the number of subaccounts to return in a single request, and supports a maximum value of 10,000.
+
+To start a paging request, add the query parameter `?cursor=initial` to your Subaccounts API request. This will fetch the first page of results. If cursor is not provided, then the default value of `initial` is applied.
+In the response body, there are three main keys: `results`, `total_count` and `links`. `results` is an array which contains the page of JSON subaccounts that matched the request,
```

Comment:
"Paginated response bodies have three main keys:"

Reply:
Actually, could we try this paragraph as just a data structure like the subaccounts object